cmd/edenPod.go

```diff
@@ -302,6 +305,39 @@ var podPurgeCmd = &cobra.Command{
 					app.Purge = &config.InstanceOpsCmd{Counter: 0}
 				}
 				app.Purge.Counter++
+				volumeConfigs := dev.GetVolumes()
+				for i, oldUUID := range volumeConfigs {
+					v, err := ctrl.GetVolume(oldUUID)
+					if err != nil {
+						log.Fatal(err)
+					}
+					if explicitVolumes {
+						skip := true
+						for _, el := range volumesToPurge {
+							if el == v.DisplayName {
+								skip = false
+								break
+							}
+						}
+						if skip {
+							continue
+						}
+					}
+					newUUID, err := uuid.NewV4()
+					if err != nil {
+						log.Fatal(err)
+					}
+					//update uuid to fire purge
+					v.Uuid = newUUID.String()
+					volumeConfigs[i] = newUUID.String()
```

Comment:
I think the purgeCmd counter increasing was sufficient to cause the purge. Has something changed in how EVE behaves to also require a new UUID and/or generation counter for the volume+volumeref?